### PR TITLE
Add service nodes for docs helpers UI

### PR DESCRIPTION
**Pull request by akegaviar**
Added the service nodes sections for each of the consortium protocols.
It's also a good place to have them in case we change stuff / add more functionality
around service nodes in the future

Closes [UI #1938](https://github.com/chainstack/ui/issues/1938)

### CODE REVIEW

File: docs/operations/corda/service-nodes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+# Service nodes
+
+A Corda network is deployed with the following service nodes:
+
+* Notary — a [notary node](/blockchains/corda#notary-service).
+* Network Map Service — a [network map service node](/blockchains/corda#network-map-service).
+
+The service nodes are provided for free.
```
Comment (easeev) [inline docs/operations/corda/service-nodes.md:16]: I suggest to remove it from the platform docs

File: docs/operations/fabric/service-nodes.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    content: Learn what service nodes come with the Hyperledger Fabric deployment on Chainstack.
+  - name: keywords
+    content: hyperledger fabric node service explorer orderer
+---
+
+# Service nodes
+
+A Hyperledger Fabric network is deployed with the following service nodes:
+
+* Explorer — an explorer node.
```
Comment (easeev) [inline docs/operations/fabric/service-nodes.md:13]: We should explain that each organization who participates in the network has its own explorer

Related to https://github.com/chainstack/ui/issues/1974

File: docs/operations/multichain/service-nodes.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+meta:
+  - name: description
+    content: Learn what service nodes come with the MultiChain deployment on Chainstack.
+  - name: keywords
+    content: multichain node service explorer
+---
+
+# Service nodes
+
+A MultiChain network is deployed with the following service node:
```
Comment (easeev) [inline docs/operations/multichain/service-nodes.md:11]: I think it's worth explaining a bit more about explorers and linking to https://github.com/chainstack/multichain-explorer

File: docs/operations/fabric/service-nodes.md (hunk 2 above)
Comment (easeev) [reply docs/operations/fabric/service-nodes.md:13]: Also can link to https://www.hyperledger.org/use/explorer

File: docs/operations/quorum/service-nodes.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+meta:
+  - name: description
+    content: Learn what service nodes come with the Quorum deployment on Chainstack.
+  - name: keywords
+    content: quorum node service explorer
+---
+
+# Service nodes
+
+A Quorum network is deployed with the following service node:
```
Comment (easeev) [inline docs/operations/quorum/service-nodes.md:11]: Same as above, can explain that we're running https://github.com/blk-io/epirus-free

File: docs/operations/corda/service-nodes.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  - name: keywords
+    content: corda node service notary map
+---
+
+# Service nodes
+
+A Corda network is deployed with the following service nodes:
+
+* Notary — a [notary node](/blockchains/corda#notary-service).
+* Network Map Service — a [network map service node](/blockchains/corda#network-map-service).
```
Comment (easeev) [inline docs/operations/corda/service-nodes.md:14]: Worth mentioning that we use Cordite Network Map Service https://gitlab.com/cordite/network-map-service

File: docs/operations/corda/service-nodes.md (hunk 1 above)
Comment (akegaviar) [reply docs/operations/corda/service-nodes.md]: done

File: docs/operations/fabric/service-nodes.md (hunk 2 above)
Comment (akegaviar) [reply docs/operations/fabric/service-nodes.md]: done

File: docs/operations/multichain/service-nodes.md (hunk 3 above)
Comment (akegaviar) [reply docs/operations/multichain/service-nodes.md:11]: done

File: docs/operations/quorum/service-nodes.md (hunk 4 above)
Comment (akegaviar) [reply docs/operations/quorum/service-nodes.md:11]: done

File: docs/operations/corda/service-nodes.md (hunk 5 above)
Comment (akegaviar) [reply docs/operations/corda/service-nodes.md]: done

File: docs/operations/fabric/service-nodes.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    content: Learn what service nodes come with the Hyperledger Fabric deployment on Chainstack.
+  - name: keywords
+    content: hyperledger fabric node service explorer orderer
+---
+
+# Service nodes
+
+A Hyperledger Fabric network is deployed with the following service nodes:
+
+* Explorer — an explorer node using the [Hyperledger Explorer](https://www.hyperledger.org/use/explorer) tool. Each organization participating in a Hyperledger Fabric network has its own explorer node deployed.
```
Comment (easeev) [inline docs/operations/fabric/service-nodes.md:13]: >an explorer node using the Hyperledger Explorer tool

Sounds really awkward IMO

File: docs/operations/corda/service-nodes.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  - name: keywords
+    content: corda node service notary map
+---
+
+# Service nodes
+
+A Corda network is deployed with the following service nodes:
+
+* Notary — a [notary node](/blockchains/corda#notary-service).
+* Network Map Service — a [network map service node](/blockchains/corda#network-map-service) using the [Cordite Network Map Service](https://gitlab.com/cordite/network-map-service) tool.
```
Comment (easeev) [inline docs/operations/corda/service-nodes.md:14]: >a network map service node Cordite Network Map Service tool

Sounds really awkward as well

Comment (akegaviar) [reply docs/operations/corda/service-nodes.md]: fixed

File: docs/operations/fabric/service-nodes.md (hunk 6 above)
Comment (akegaviar) [reply docs/operations/fabric/service-nodes.md]: fixedx

File: docs/operations/corda/service-nodes.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+  - name: keywords
+    content: corda node service notary map
+---
+
+# Service nodes
+
+A Corda network is deployed with the following service nodes:
+
+* Notary — a [notary node](/blockchains/corda#notary-service).
+* Network Map Service — a [network map service node](/blockchains/corda#network-map-service) running [Cordite](https://gitlab.com/cordite/network-map-service).
```
Comment (easeev) [inline docs/operations/corda/service-nodes.md:14]:
```suggestion
* Network Map Service — a [network map service node](/blockchains/corda#network-map-service) running [Cordite Network Map Service](https://gitlab.com/cordite/network-map-service).
```

File: docs/operations/fabric/service-nodes.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    content: Learn what service nodes come with the Hyperledger Fabric deployment on Chainstack.
+  - name: keywords
+    content: hyperledger fabric node service explorer orderer
+---
+
+# Service nodes
+
+A Hyperledger Fabric network is deployed with the following service nodes:
+
+* Explorer — a node running [Hyperledger Explorer](https://www.hyperledger.org/use/explorer) to scan and view the network. Each organization participating in a Hyperledger Fabric network has its own explorer node deployed.
```
Comment (easeev) [inline docs/operations/fabric/service-nodes.md:13]:
```suggestion
* Explorer — a node running [Hyperledger Explorer](https://www.hyperledger.org/use/explorer). Each organization participating in a Hyperledger Fabric network has its own explorer node deployed.
```

**Pull request merged**